src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -107,7 +107,7 @@ for further details.
 .breadcrumbContainer {
 	display: flex;
 	justify-content: flex-start;
-	margin-bottom: 32px;
+	margin-bottom: 24px;
```

Comment:
I confirmed with @braydenlove that this update should be global to all sidebar / sidecar layouts.

Reply:
Thanks, also got confirmation from @heitzke that this is a good global refinement.